pages/views/InstallmentCalculator.jsx

```diff
@@ -0,0 +1,182 @@
+
+    setInterestA(interestA);
+    setInterestB(interestB);
+
+    interestA === interestB ? setMsgResult('is eaqul.')
+    :interestA > interestB ? setMsgResult('A is better!')
+    :setMsgResult('B is better!');
+
+    console.log(windowWidth)
+    windowWidth < 767.98 ? setIsModalOpen(true) : setIsModalOpen(false)
+  }
+
+  return (
+    <ConfigProvider
+      theme={{
+        components: {
+          InputNumber: {
+            colorPrimaryHover: '#beed54',
+            colorFillAlter: 'white'
+          },
+          Select: {
+            colorPrimaryHover: '#beed54'
+          },
+          Button: {
+            colorPrimaryActive: '#beed54',
+            colorPrimaryHover: '#beed54',
+            colorText: '#beed54',
+          },
+          Modal: {
+            contentBg: '#333333',
+            headerBg: '#333333',
+            titleColor: 'white',
+            colorText: 'white',
+            colorIcon: 'white'
+          }
+        },
+      }}
+    >
+      <Row gutter={{ xs: 8, md: 16 }}>
+        <Col 
+          xs={{
+            span: 24,
+            offset: 0,
+          }}
+          md={{
+            span: 12,
+            offset: 4,
+          }}
+        >
+          <h2 className={styles.title}>Calculator</h2>
+          <div style={{marginBottom: '25px'}}>
+            <Space direction="vertical" size="small">
+              <h3>Amount :</h3>
+              <InputNumber
+                addonBefore="$"
+                defaultValue={1000}
+                formatter={(value) => value.replace(/\B(?=(\d{3})+(?!\d))/g, ',')}
+                style={inputStyle}
+                onChange={amountChange}
+              />
+            </Space>
+          </div>
+          <Row gutter={[0, {xs: 24, md: 0}]}>
+            <Col xs={24} md={12}>
+              <h4>Plan A</h4>
+              <InstallmentInput planData={planAData}/>
+            </Col>
+            <Col xs={24} md={12}>
+              <h4>Plan B</h4>
+              <InstallmentInput planData={planBData}/>
+            </Col>
+          </Row>
+          <div>
+            <Button onClick={updateInterest}>Calculate</Button>
+          </div>
+        </Col>
+        <Col xs={24} md={6} className={styles.divResult}>
+          <h2 className={styles.title}>Result</h2>
+          <div style={divResult}>
+            <h3>Plan {msgResult}</h3>
+            <h3>
+              {interestA === interestB ? ''
+              :interestA > interestB ? '$'+ interestA
+              :'$' + interestB}
+            </h3>
+          </div>
+          <div>
+            <h4>Detail</h4>
+            <p>Plan A Interest : ${interestA}</p>
+            <p>Plan B Interest : ${interestB}</p>
+            <p>All Weather Portfolio APR : {(portfolioApr).toFixed(2)}%</p>
+            <p>Formula : (Amount / Installment) * (1 + Installment) * Installment / 2 * (All Weather Portfolio APR - Interest Rate ) / 12 </p>
+          </div>
+        </Col>
```

Comment:
really like this block, way better than what I expected 💯 

![Uploading Screenshot 2023-11-16 at 8.44.57 AM.png…]()

Reply:
You can switch to mobile view, it is hidden. When you click the button, it will show the results in a popup.
Do you like this design?